include/VerificationValidation.h

```diff
@@ -66,9 +67,9 @@ namespace VerificationValidation {
         static void searchDBNotFoundErrors(Result* r, const QString& currentLine);
         static void searchFinalDefense(Result* r);
 
-        static Result* lc(const QString* terminalOutput);
+        static Result* lc(const QString* cmd, const QString* terminalOutput);
```

Comment:
kind of nit-picky, but it'd be nice if this was "const QString& cmd", so whenever calling:
Result* r = Parser::lc(cmd, terminalOutput);
as opposed to 
Result* r = Parser::lc(&cmd, terminalOutput);

plus adds uniformity in calls for future devs